examples/medplum-fhircast-demo/src/components/Subscriber.tsx

```diff
@@ -0,0 +1,179 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { BASE_URL } from '../config';
+import { useClientId, useHubWSConnection } from '../hooks';
+import { FHIRcastMessagePayload, isWebSocketMessage, serializeHubSubscriptionRequest } from '../utils';
+import TopicLoader from './TopicLoader';
+
+type FhirCastMessageDisplayProps = {
+  eventNo: number;
+  message: object;
+};
+
+function FhirCastMessageDisplay(props: FhirCastMessageDisplayProps): JSX.Element {
+  return (
+    <div style={{ paddingBottom: 15 }}>
+      <div
+        className="fhir-cast-message"
+        style={{
+          paddingLeft: 30,
+          paddingRight: 30,
+          paddingTop: 20,
+          paddingBottom: 20,
+          borderRadius: 10,
+          textAlign: 'left',
+        }}
+      >
+        <h3>Event No. {props.eventNo}</h3>
+        <pre>{JSON.stringify(props.message, null, 2) ?? ''}</pre>
```

Comment:
Will `props.message` ever be `undefined`?  If not, I think we can remove the `?? ''`